src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import useToggle from '../Toggle/useToggle';
+
+const decodeHTML = str => {
+  const replacedParts = {
+    '&quot;': '"',
+    '&#39;': "'",
+    '&amp;': '&',
+    '&lt;': '<',
+    '&gt;': '>',
+  };
+  const replacementsRegex = new RegExp(
+    Object.keys(replacedParts).join('|'),
+    'gi',
+  );
+  return str.replace(replacementsRegex, match => replacedParts[match]);
+};
+
+const IncludeContainer = ({ html, type }) => {
+  const { enabled } = useToggle('include');
+
+  const supportedTypes = {
+    idt2: 'idt2',
+    vj: 'vj',
+  };
+
+  const shouldNotRenderInclude = !enabled || !html || !supportedTypes[type];
+
+  if (shouldNotRenderInclude) {
+    return null;
+  }
+
+  return (
+    <GridItemConstrainedMedium>
+      <div
```

Comment:
@greenc05 Do we need to have anything a11y specific for include containers?

Reply:
not at the moment @HarveyPeachey